app/src/main/java/com/android/unio/ui/explore/Explore.kt

```diff
@@ -0,0 +1,178 @@
+package com.android.unio.ui.explore
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.material3.Icon
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.android.unio.model.association.Association
+
+@Composable
+fun ExploreScreen() {
+  Scaffold(
+      modifier = Modifier.testTag("exploreScreen"),
+      /** Here, we should have the bottom navigation bar. */
```

Comment:
We should maybe have the `ExploreScreen` composable represent only the Explore content (without the bottom bar), and the actual BottomBar is at a higher abstraction level to prevent code duplication.

Reply:
Sure, also would make it more readable. Will be done when the navigation system is available. Or we can merge this and refactor later today.

Reply:
Done.